services/frontend-service/pkg/cmd/server.go

```diff
@@ -62,15 +63,19 @@ func RunServer() {
 		grpcServerLogger := logger.FromContext(ctx).Named("grpc_server")
 
 		gsrv := grpc.NewServer(
-			grpc.StreamInterceptor(
+			grpc.ChainStreamInterceptor(
```

Comment:
The backend service has an option to disable tracing. Should the frontend service have the same?

Reply:
Thanks. I implemented this in https://github.com/freiheit-com/kuberpult/pull/198/commits/aeb47984c701a5b52b5b9215235dbb437a967bc4 and cleaned up a little.